<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Massimiliano Torromeo</author>
 <class>SQLAntaresiaWindow</class>
 <widget class="QMainWindow" name="SQLAntaresiaWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">SQL Antaresia</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/32/icons/kexi.png</normaloff>:/32/icons/kexi.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabsWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAboutSQLAntaresia"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionShowDatabases"/>
    <addaction name="actionShowStatusbar"/>
    <addaction name="separator"/>
    <addaction name="actionConfigureSQLAntaresia"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Settings"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>180</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Servers</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QTreeView" name="treeView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionRefreshTableList">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16/icons/view-refresh.png</normaloff>:/16/icons/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh Table List</string>
   </property>
  </action>
  <action name="actionRefreshDatabases">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16/icons/view-refresh.png</normaloff>:/16/icons/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh Databases</string>
   </property>
  </action>
  <action name="actionDropTable">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16/icons/edit-delete.png</normaloff>:/16/icons/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Drop Table</string>
   </property>
  </action>
  <action name="actionCloseTab">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16/icons/tab-close.png</normaloff>:/16/icons/tab-close.png</iconset>
   </property>
   <property name="text">
    <string>Close Tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionShowStatusbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show St&amp;atusbar</string>
   </property>
  </action>
  <action name="actionConfigureSQLAntaresia">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16/icons/configure.png</normaloff>:/16/icons/configure.png</iconset>
   </property>
   <property name="text">
    <string>Configure &amp;SQL Antaresia</string>
   </property>
  </action>
  <action name="actionConfigureConnections">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16/icons/configure.png</normaloff>:/16/icons/configure.png</iconset>
   </property>
   <property name="text">
    <string>Configure &amp;Connections</string>
   </property>
  </action>
  <action name="actionAboutSQLAntaresia">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/32/icons/kexi.png</normaloff>:/32/icons/kexi.png</iconset>
   </property>
   <property name="text">
    <string>About SQL Antaresia</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/64/icons/qtlogo.png</normaloff>:/64/icons/qtlogo.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16/icons/application-exit.png</normaloff>:/16/icons/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionShowDatabases">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;Databases</string>
   </property>
  </action>
  <action name="actionDropDatabase">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16/icons/edit-delete.png</normaloff>:/16/icons/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Drop Database</string>
   </property>
   <property name="toolTip">
    <string>Drop Database</string>
   </property>
  </action>
  <action name="actionReconnect">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16/icons/network-connect.png</normaloff>:/16/icons/network-connect.png</iconset>
   </property>
   <property name="text">
    <string>Re&amp;connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16/icons/network-disconnect.png</normaloff>:/16/icons/network-disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
  </action>
  <action name="actionTruncateTable">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16/icons/edit-delete.png</normaloff>:/16/icons/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Truncate Table</string>
   </property>
  </action>
  <action name="actionNewQueryTab">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16/icons/tab-new.png</normaloff>:/16/icons/tab-new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New Query Tab</string>
   </property>
  </action>
  <action name="actionOptimizeTable">
   <property name="text">
    <string>Optimize Table</string>
   </property>
  </action>
  <action name="actionRepairTable">
   <property name="text">
    <string>Repair Table</string>
   </property>
  </action>
  <action name="actionGo">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16/icons/system-run.png</normaloff>:/16/icons/system-run.png</iconset>
   </property>
   <property name="text">
    <string>Go</string>
   </property>
  </action>
  <action name="actionShowCreate">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16/icons/document-properties.png</normaloff>:/16/icons/document-properties.png</iconset>
   </property>
   <property name="text">
    <string>Show Create</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16/icons/view-refresh.png</normaloff>:/16/icons/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Refresh</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionConfigureConnection">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16/icons/configure.png</normaloff>:/16/icons/configure.png</iconset>
   </property>
   <property name="text">
    <string>Configure server</string>
   </property>
   <property name="iconText">
    <string>Configure server</string>
   </property>
   <property name="toolTip">
    <string>Configure server</string>
   </property>
  </action>
  <action name="actionRemoveConnection">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16/icons/edit-delete.png</normaloff>:/16/icons/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Remove server</string>
   </property>
   <property name="iconText">
    <string>Remove server</string>
   </property>
   <property name="toolTip">
    <string>Remove server</string>
   </property>
  </action>
  <action name="actionAddConnection">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16/icons/list-add.png</normaloff>:/16/icons/list-add.png</iconset>
   </property>
   <property name="text">
    <string>Add new server</string>
   </property>
   <property name="iconText">
    <string>Add new server</string>
   </property>
   <property name="toolTip">
    <string>Add new server</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>treeView</tabstop>
  <tabstop>tabsWidget</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>SQLAntaresiaWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShowStatusbar</sender>
   <signal>triggered(bool)</signal>
   <receiver>statusBar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>525</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShowDatabases</sender>
   <signal>triggered(bool)</signal>
   <receiver>dockWidget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockWidget</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionShowDatabases</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
